packages/server/src/automations/steps/filter.ts

```diff
@@ -73,7 +73,12 @@ export async function run({ inputs }: AutomationStepInput) {
   try {
     let { field, condition, value } = inputs
     // coerce types so that we can use them
-    if (!isNaN(value) && !isNaN(field)) {
+    if (
+      !isNaN(value) &&
+      !isNaN(field) &&
+      typeof field !== "boolean" &&
```

Comment:
Shouldn't this be checking if the `typeof field === "string"`?

Reply:
I think we want to specifically check for not boolean here